_config.yml

```diff
@@ -116,8 +116,8 @@ defaults:
 #   - dest is the destination path within the working tree
 #   - src is a list of glob source paths within the remote repository
 fetch-remote:
-  - repo: "https://github.com/docker/cli"
-    ref: "20.10"
+  - repo: "https://github.com/crazy-max/docker-cli" # FIXME: Use https://github.com/docker/cli
+    ref: "20.10-dockerfile-ref-moved" # FIXME: Use 20.10
```

Comment:
I have started to change links to the new location in my fork https://github.com/docker/cli/compare/20.10...crazy-max:20.10-dockerfile-ref-moved

One missing change in distribution repo to do:

```
 #16 173.1 - ./_site/registry/spec/deprecated-schema-v1/index.html
#16 173.1   *  linking to internal hash #from that does not exist (line 141)
#16 173.1      <a href="https://docs.docker.com/engine/reference/builder/#from">Dockerfile reference</a>
```